src/config/keys.ts

```diff
@@ -117,91 +251,25 @@ export const buildFileContentKey = ({
   replyUrl?: boolean;
 }) => [ITEMS_CONTEXT, 'file', id, replyUrl ? 'url' : 'blob'];
 
-export const ITEM_FLAGS_CONTEXT = 'itemFlags';
 export const buildItemFlagsKey = (id: UUID) => [ITEMS_CONTEXT, 'flags', id];
 
 export const buildCategoryKey = (id: UUID) => ['category', id];
 export const buildCategoriesKey = (typeId?: UUID[]) => [
   'categories',
   hashItemsIds(typeId),
 ];
-export const buildItemCategoriesKey = (id?: UUID) => [
-  ITEMS_CONTEXT,
-  'categories',
-  id,
-];
+
 export const buildItemsByCategoriesKey = (ids: UUID[]) => [
   'itemsInCategories',
   hashItemsIds(ids),
 ];
```

Comment:
Is it still used? Remove?

Reply:
Yes it is used in a hooks that does not seem to be used ... So i just moved it the item keys.